docs/variables.md

```diff
@@ -88,6 +88,11 @@ Targeting service crn:v1:bluemix:public:power-iaas:tor01:a/65b64c1f1c2XXXXX:4a1f
 1) ocp-net
 #? 1
 - You have answered: ocp-net
+[question] > Select the system type to use for cluster nodes:
+1) e980
+2) s922
+#? 2
```

Comment:
E880 is also part of some powervs regions.. how are we gonna handle it if user selects 'e980' there?
@bpradipt

Reply:
https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-about-virtual-server

Reply:
@yussufsh had discussed regarding the same with @bpradipt 
Dallas and Washington has e880 - did not filter it out as if user selects the same in those region we expect it to work in theoretical terms (if this is something that needs to be avoided let me know, I can filter to exclude it)

Reply:
You can validate the theory or fix it for the given regions.

Reply:
Had asked Aishwarya to create cluster with e880 -- seems to be working fine no issues there. (will ask validation team to do some testing)